marlowe-dashboard-client/src/Template/View.purs

```diff
@@ -58,7 +58,7 @@ contractSetupScreen walletLibrary currentSlot state =
     payIsAccessible = termsAreAccessible && templateContentIsValid templateContent slotContentStrings currentSlot
   in
     div
-      [ classNames [ "grid", "grid-rows-contract-setup", "max-h-full", "overflow-hidden" ] ]
+      [ classNames [ "grid", "grid-rows-contract-setup", "h-full", "overflow-hidden" ] ]
```

Comment:
This fixes an unrelated visual bug I noticed at the same time. It didn't make a difference before I added the custom dropdowns to the role inputs - but with a max height, the dropdown was sometimes being cut off from view.